dune

```diff
@@ -791,6 +791,10 @@
 (install
  (files
   (compilerlibs/META as compiler-libs/META)
+  (otherlibs/stdlib_upstream_compatible/META as stdlib_upstream_compatible/META)
```

Comment:
Should these go in the dune files for each lib?

Reply:
Afaik, if a dune file has an `install` stanza, it won't automatically generate the list of files to install from `.ml`s. And I'd rather avoid unnecessarily listing all of those.

Reply:
`stdlib_alpha/dune` appears to already have an `install` block, though?